projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/JobImageBuilder.java

```diff
@@ -38,6 +38,9 @@ public class JobImageBuilder {
   @Value("${datajobs.git.url}")
   private String gitRepo;
 
+  @Value("${datajobs.aws.assumeIAMRole:}")
+  private boolean useRoleCredentials;
```

Comment:
I'd encapsulate this behind AWSServiceAccountCredentialsService and it will return credentials, if assumeIAMRole is false it would return session token as empty.

Reply:
Also name both the property (assumeIAMRole) and the field name (useRoleCredentials) with the same name please.